pandas-stubs/core/series.pyi

```diff
@@ -1755,10 +1757,14 @@ class TimedeltaSeries(Series[Timedelta]):
     @overload
     def __add__(self, other: Timedelta | np.timedelta64) -> TimedeltaSeries: ...
     def __radd__(self, pther: Timestamp | TimestampSeries) -> TimestampSeries: ...  # type: ignore[override]
+    @overload
     def __mul__(self, other: num) -> TimedeltaSeries: ...  # type: ignore[override]
     def __sub__(  # type: ignore[override]
         self, other: Timedelta | TimedeltaSeries | TimedeltaIndex | np.timedelta64
     ) -> TimedeltaSeries: ...
+    def __truediv__(self, other: TimedeltaSeries | np.timedelta64) -> Series[float]: ...  # type: ignore[override]
+    @overload
+    def __mul__(self, other: TimestampSeries | np.timedelta64) -> Never: ...  # type: ignore[override]
```

Comment:
you need to put all the definitions of the same methods that have overloads together.  Move lines 1766-1767 to after line 1761

Reply:
thanks sir

Reply:
> you need to put all the definitions of the same methods that have overloads together. Move lines 1766-1767 to after line 1761

done sir

Reply:
> `error: Signature of "__mul__" incompatible with supertype "Series" [override]` this is the last mypy error it doesn't go if I add `# type: ignore[override]` from line number 1760

It's OK to put in the `# type: ignore[override]` in this case.

Reply:
> > `error: Signature of "__mul__" incompatible with supertype "Series" [override]` this is the last mypy error it doesn't go if I add `# type: ignore[override]` from line number 1760
> 
> It's OK to put in the `# type: ignore[override]` in this case.

then it shows this error `error: Unused "type: ignore" comment`

Reply:
check the line numbers that the error is reported on

Reply:
on  the same line where the incompatible with supertype error is